src/client/transforms.tsx

```diff
@@ -78,45 +78,13 @@ const mimeTypeToImport: TransformData[] = [
         const module = await import(/* webpackChunkName: "nteract_transforms" */ '@nteract/transforms');
         return module.SVGTransform;
     }),
-    new TransformData('image/png', async () => {
-        const module = await import(/* webpackChunkName: "nteract_transforms" */ '@nteract/transforms');
-        return module.PNGTransform;
-    }),
-    new TransformData('image/gif', async () => {
-        const module = await import(/* webpackChunkName: "nteract_transforms" */ '@nteract/transforms');
-        return module.GIFTransform;
-    }),
-    new TransformData('image/jpeg', async () => {
-        const module = await import(/* webpackChunkName: "nteract_transforms" */ '@nteract/transforms');
-        return module.JPEGTransform;
-    }),
-    new TransformData('application/json', async () => {
```

Comment:
These seem odd? Does VS code handle them by itself?

Reply:
Or maybe we never registered these in VS code

Reply:
> oes VS code handle them by itself?

Yes

> e never registered these in VS code

Yes